### PR TITLE
Add sections for evals and use cases; cite evals appropriately; update methods considered table

### PR DESCRIPTION
**Pull request by dhh1128**
***
<a href="https://pr-preview.s3.amazonaws.com/dhh1128/did-rubric/pull/50.html" title="Last updated on Nov 1, 2021, 9:21 AM UTC (a92975a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-rubric/50/21f9ecd...dhh1128:a92975a.html" title="Last updated on Nov 1, 2021, 9:21 AM UTC (a92975a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -412,84 +412,76 @@ <h3>The architecture</h3>
...
-          <th>
-            Evaluation date
-          </th>
-          <td>
-            2020-01-03
-          </td>
+        <tr id="eval-2">
+          <td>eval-2</td>
+          <td>DID Method Rubric</td>
+          <td>Evaluations written by Daniel Hardman, to illusfor this rubric. Not explicitly funded.</td>
```
Comment (TallTed) [inline index.html:436]:
```suggestion
          <td>Evaluations written by Daniel Hardman, to illustrate this rubric. Not explicitly funded.</td>
```

File: index.html [hunk 2]
```diff
@@ -412,84 +412,76 @@ <h3>The architecture</h3>
...
-            <p>
-              <strong>Long term verifiable credentials.</strong> The use of DIDs as subject
-              identifiers for long term (life-long) verifiable credentials such as earned
-              academic degrees.
-            </p>
+        <tr id="eval-3">
+          <td>eval-3</td>
+          <td>"A Rubric for Decentralization of DID Methods". Joe Andrieu, Shannon Appelcline,
+            Amy Guy, Joachim Lohkamp, Drummond Reed, Markus Sabadello, Oliver Terbu. <cite>Rebooting
+              the Web of Trust</cite>. Published 2020-07020. <a
```
Comment (TallTed) [inline index.html:442]:
```suggestion
              the Web of Trust</cite>. Published 2020-07-20. <a
```

File: index.html [hunk 3]
```diff
@@ -412,84 +412,76 @@ <h3>The architecture</h3>
...
+              href="https://github.com/WebOfTrustInfo/rwot9-prague/blob/master/final-documents/decentralization-rubric.pdf">
+              https://github.com/WebOfTrustInfo/rwot9-prague/blob/master/final-documents/decentralization-rubric.pdf</a>.
           </td>
+          <td>The original DID Rubric paper from Rebooting the Web of Trust. Not explicitly funded.
+            Not explicitly funded.</td>
```
Comment (TallTed) [inline index.html:447]:
```suggestion
          <td>The original DID Rubric paper from Rebooting the Web of Trust. Not explicitly funded.</td>
```

**Comment by jandrieu** [comment]
@dhh1128 Looks like we have an error in the CI/CD tooling. It is likely because of a couple of missing <ol> tags (at least I had that problem on the registry PR and your PR likely has the same cruft). You can probably identify the error location by finding a </p> tag followed by a <li> tag. At least that helped with the instances I had fixed.

Alternatively, merge in #49 and pull main back into your PR. That would probably clean it up with less likelihood of a merge conflict.

**Comment by TallTed** [comment]
@jandrieu -- You need to codefence the HTML tags in [your comment](https://github.com/w3c/did-rubric/pull/50#issuecomment-923191375), to make them visible to readers.

**Comment by rxgrant** [comment]
This is stumping the editors and blocking other merges and action right now.

File: index.html [hunk 4]
```diff
@@ -593,15 +584,175 @@ <h3>Comparative evaluation report header</h3>
...
+              JLINC DID Method Specification</a>
+          </td>
+          <td>
+            JLincLabs
+          </td>
+          <td>
+            Register DIDs over the JLinc protocol for sharing data with terms and conditions.
+          </td>
+        </tr>
+        Did:jlinc
```
Comment (rxgrant) [inline index.html:752]: Did:jlinc here looks wrong

File: index.html [hunk 5]
```diff
@@ -593,15 +584,175 @@ <h3>Comparative evaluation report header</h3>
...
+          <td>
+            Gabe Choen
+          </td>
+          <td>
+            Publish a DID via Twitter feed.
+          </td>
+        </tr>
+        <tr id="did:twit">
+          <td>
+            did:twit
```
Comment (rxgrant) [inline index.html:709]: did:twit duplicated.  breaking build.

**Review by rxgrant — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (TallTed) [inline index.html:709]:
```suggestion
        <tr id="did:pkh">
          <td>
            did:pkh
```

File: index.html [hunk 6]
```diff
@@ -593,15 +584,175 @@ <h3>Comparative evaluation report header</h3>
...
+          <td>
+            JLincLabs
+          </td>
+          <td>
+            Register DIDs over the JLinc protocol for sharing data with terms and conditions.
+          </td>
+        </tr>
+        Did:jlinc
+
       </table>
```
Comment (TallTed) [inline index.html:754]:
```suggestion
        </tr>
      </table>
```

**Review by TallTed — CHANGES_REQUESTED** [review]
I believe my two suggested changes will resolve the build error, and remove a tiny errant snippet of text.

File: index.html (hunk 6 above)
Comment (dhh1128) [reply index.html]: Updated and credit you. Thanks.

File: index.html (hunk 5 above)
Comment (dhh1128) [reply index.html]: Updated and credited you.

File: index.html (hunk 4 above)
Comment (dhh1128) [reply index.html]: Agreed. Fixed. thanks.

File: index.html (hunk 5 above)
Comment (dhh1128) [reply index.html]: Updated and fixed.

**Review by rxgrant — APPROVED** [review]
changes look good.

**Comment by jandrieu** [comment]
This looks good. Merging.

**Comment by jandrieu** [comment]
Looks like the edits from this merge got smushed.

@rxgrant Let's review and figure out how to get the evaluation citations and related metadata back into the document.

**Comment by jandrieu** [comment]
@rxgrant and I found that this was a synchronization problem between the main branch, the github.io version, and the /TR version.

We removed the hardcoded publication date (August 26) from the index.html and that eventually got us an updated github.io page, but the TR has not yet updated. 

We're going to give it another day to see if the TR updates, but we are also baffled as to why the date in the html would prevent github pages from rendering a new .io page.  When we removed the date, it didn't just give us the correct date, it actually pulled in the latest version. How and why that happens seems to be buried in the github pages configuration.

@iherman Can you advise about how we should be doing this? I think we aren't quite configured right for auto publishing and I expect our hack of removing the date may not be the "right" way.

**Comment by iherman** [comment]
> @iherman Can you advise about how we should be doing this? I think we aren't quite configured right for auto publishing and I expect our hack of removing the date may not be the "right" way.

There seem to be some system problems. I have updated the github action script but that did not solve all the issues. See my [mail](https://lists.w3.org/Archives/Public/spec-prod/2021OctDec/0056.html) sent to the powers to be (I cc-d you because I was not sure that you can access that archive).

Sigh...

**Pull request merged**